SPECS/python-poetry-core/python-poetry-core.spec

```diff
@@ -0,0 +1,130 @@
+%global pypi_name poetry-core
+
+Summary:        Poetry PEP 517 Build Backend
+Name:           python-poetry-core
+Version:        1.0.7
```

Comment:
Would love to see this bumped to 1.2.0- the bug fix delta seems pretty significant here.